massa-api/src/lib.rs

```diff
@@ -133,6 +133,17 @@ pub trait Endpoints {
     #[rpc(name = "ban")]
     fn ban(&self, _: Vec<IpAddr>) -> BoxFuture<Result<(), ApiError>>;
 
+    /// whitelist given IP address.
+    /// No confirmation to expect.
+    #[rpc(name = "node_whitelist")]
+    fn node_whitelist(&self, _: Vec<IpAddr>) -> BoxFuture<Result<(), ApiError>>;
```

Comment:
List the IpAddress in the whitelist in the node?

Reply:
Can you write that response? It's lot better :wink:

Reply:
you mean in the help ? I'm writing the api doc here #2318

Reply:
no, right here in the code
```suggestion
    /// whitelist given IP address.
    /// No confirmation to expect.
    /// Note: If the ip was unknown it adds it to the known peers, otherwise it updates the peer type
    #[rpc(name = "node_whitelist")]
    fn node_whitelist(&self, _: Vec<IpAddr>) -> BoxFuture<Result<(), ApiError>>;
```